docs/integrations/containers-orchestration/vmware-tanzu-application-service.md

```diff
@@ -0,0 +1,158 @@
+---
+id: vmware-tanzu-application-service
+title: VMware Tanzu Application Service
+sidebar_label: VMware Tanzu Application Service
+description: This documentation describes the Sumo Logic Nozzle for VMware Tanzu tile.
+---
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/vmware-tanzu.png')} alt="VMware Tanzu Application Service" width="50" />
+
+## <a id='nozzle-overview'></a> Overview of Sumo Logic Nozzle for VMware Tanzu
+
+Sumo Logic Nozzle for VMware Tanzu can help you:
```

Comment:
@akhil-sumologic , our docs style is to omit the "overview" header and go straight into the overview description

```suggestion
The Sumo Logic Nozzle for VMware Tanzu can help you:
```

Reply:
> @akhil-sumologic , our docs style is to omit the "overview" header and go straight into the overview description

Addressed